spec.html

```diff
@@ -39904,6 +39904,57 @@ <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
         1. Return ? ForIn/OfBodyEvaluation(|BindingIdentifier|, |Statement|, _keyResult_, ~enumerate~, ~varBinding~, _labelSet_).
       </emu-alg>
     </emu-annex>
+
+    <emu-annex id="sec-html-document-dot-all-exotic-object">
+      <h1>HTML Document Dot All Exotic Object</h1>
+      <p>An <dfn>HTML document dot all exotic object</dfn> is an exotic object that behaves like undefined in the <emu-xref href="#sec-toboolean">ToBoolean</emu-xref> and <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> abstract operations and when used as an operand for the <emu-xref href="#sec-typeof-operator">typeof operator</emu-xref>. HTML document dot all exotic objects have all the ordinary object internal methods and slots.</p>
```

Comment:
Maybe one way to do this would be to state somehow that any object (well, one precise object) can be "marked" as a HTML document dot all exotic object, not that it "is" a HTML dot all exotic object. That way it could be a "legacy platform exotic object" (= using various internal methods defined by Web IDL) plus be marked as a HTML document dot all exotic object, at the same time.